Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/HolographicButton.prefab

```diff
@@ -389,8 +389,9 @@ MonoBehaviour:
   Dimensions: 1
   CanSelect: 1
   CanDeselect: 1
-  VoiceCommand: 
+  VoiceCommand: Button
   RequiresFocus: 1
+  PhysicalPressOnClickTrigger: 0
```

Comment:
I noticed that the interactable has a behavior that's now quite similar to PushButton.cs:
![image](https://user-images.githubusercontent.com/168492/57797861-42724080-7700-11e9-8861-0a1f265d5f80.png)

In particular, the interactable is now supporting a push-button behavior that's the same as PressableButton.

Could you provide some explanation about why you'd like to have this behavior directly in Interactable? There is an advantage of having all the behavior stored in a single script, but a disadvantage I see is that with two pieces of code implementing the same behavior, any improvements to PressableButton will now also need to be made to Interactable.cs.

Additionally, we would need to all agree on whether we are recommending customers to use Interactable, or PressableButton.cs, or all agree / commit that either component can be used and behavior will be identical. 

Since I could see the decision of whether to implement pressable button directly in interactable could create a bit of discussion, it might make sense to pull that change out of this PR and create a separate PR just for that behavior, so we can have that discussion there. For example, one way I could see to solve this is to abstract away the push button logic into a class, and then have interactable and pressablebutton use the same underlying logic. But that might be a bigger task than you want to do in this PR alone, since you would want to work with @thalbern on this.

Reply:
That's a good idea. Things will change over time or other use cases will popup that will require changes to button tech or new components to be written, I'm all for keeping that stuff condensed.

Reply:
removed the press OnClick trigger and detecting touch start and deltas